setup.py

```diff
@@ -8,4 +8,8 @@
     author="Tom Legkov",
     author_email="tom.legkov@outlook.com",
     packages=["marine"],
+    include_package_data=True,
+    package_data={
+        "marine": [".wslibs/*.so*"],
```

Comment:
Why wildcards?

Reply:
Why would I name them explicitly? I want to package into the wheel all the `so`s in this directory